uphold/vault-assist-tool
# Add support for hedera vault

### Testing:
1. Make sure you have a funded HBAR vault
2. Make sure `vault-assist-tool` is running locally `yarn dev`
3. Open in web browser `http://localhost:3000/`
4. Select cyrpto asset: HBAR
5. Enter HBAR vault address
6. Enter destination (probably good to use your HBAR development wallet for now)
7. Put in the keys
8. Submit withdraw
9. Confirm transaction
10. Run e2e tests: `yarn e2e`
11. Manually run tests for bitcoin vault

### Note 
After you sweep the funds, it will leave some HBAR dust in the vault. This is due to the way fees work in Hedera. There isn't a "close vault" feature for HBAR, so VAT is only going to be used to sweep funds.

### Tasks
LABS-807
LABS-851
